charts/views.py

```
@@ -1,18 +1,14 @@
-from django.db.models import Count, Min, Q
-from bokeh.models import Range1d
+from django.db.models import Count, Q
 from fighthealthinsurance.models import Denial, InterestedProfessional
 from bokeh.plotting import figure
 from bokeh.embed import components
-from bokeh.models import DatetimeTickFormatter
 from bokeh.models import ColumnDataSource
 import pandas as pd
 import csv
-from django.http import HttpResponse, StreamingHttpResponse
+from django.http import HttpResponse
```

Ruff (0.8.2)

Redefinition of unused `HttpResponse` from line 1

Remove definition: `HttpResponse`

(F811)